quizcon/main/models.py

```diff
@@ -3,13 +3,70 @@
 from django.db import models
 
 
+TRIANGLE_SIDE = 4
+
+EASY = {
+    "0": 5,
+    "1": 4,
+    "2": 3,
+    "3": 1,
+    "4": 0,
+    "5": 0,
+    "6": 0,
+    "7": 0,
+    "8": 0,
+    "9": 1,
+    "10": 3,
+    "11": 4,
+    "12": 2
+    }
+MEDIUM = {
+    "0": 3,
+    "1": 2,
+    "2": 1,
+    "3": -1,
+    "4": -2,
+    "5": -2,
+    "6": -2,
+    "7": -2,
+    "8": -2,
+    "9": -1,
+    "10": 1,
+    "11": 2,
+    "12": 0
+    }
+HARD = {
+    "0": 3,
+    "1": 2,
+    "2": 1,
+    "3": -1,
+    "4": -5,
+    "5": -5,
+    "6": -5,
+    "7": -5,
+    "8": -5,
+    "9": -1,
+    "10": 1,
+    "11": 2,
+    "12": 0
+    }
+
+SCORING_SCHEMES = [
+    (1, 'Easy'),
```

Comment:
Note: The logic could be a simplified a little if SCORING_SCHEMES was 0-based. That would eliminate the need for the `-1` at Ln. 147. And, you could make `LEVELS` a 0-based array that would allow you to map the index directly to the schema.